client/src/components/ImageUpdateModal.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import {Button, Modal, Form} from 'semantic-ui-react';
+import swal from "sweetalert";
+
+const BACKEND = process.env.REACT_APP_BACKEND = "http://localhost:5000"
+
+/*
+props:
+    - modalOpen: boolean
+    - closeModal: ()
+    - isAlumni: boolean
+    - id
+*/
+export default class ImageUpdateModal extends Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            submitting: false,
+            imageFile: null
+        }
+        this.selectFile = this.selectFile.bind(this)
+        this.submit = this.submit.bind(this)
+        this.fileTypeIsImage = this.fileTypeIsImage.bind(this)
+    }
+
+    fileTypeIsImage (file) {
+        return file && (file.type === 'image/jpeg' || file.type === 'image/jpg' || file.type === 'image/png')
+    }
+
+    selectFile(e) {
+        e.preventDefault();
+        let file = e.target.files[0]
+        if (this.fileTypeIsImage(file)) {
+            this.setState({
+                imageFile: e.target.files[0]
+            })
+        } else {
+            swal({
+                title: "Whoops!",
+                text: "Please select a jpeg, jpg, or a png file",
+                icon: "warning",
+            });
+        }
+    }
+
+    submit(e) {
+        e.preventDefault()
+        this.setState({
+            submitting: true
+        }, async () => {
+            try {
+                let data = new FormData();
+                data.append('imageFile', this.state.imageFile);
+                fetch(`${BACKEND}/image/${this.props.isAlumni?`alumni` : `student`}/${this.props.id}`, {
```

Comment:
I didn't use the `makeCall` helper we have set up since we do not want to add the `application/json` Content-Type